index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <nav class="navbar">
+        <div class="log">My Logo</div>
+        <nav>
+          <a href="#" class="header-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </a>
+        </nav>
+        <div class="nav-links">
+          <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section>
+      <div class="background-img">
+        <div class="row">
+          <div class="main">
+            <h1 class="main-description">
+              <div>I'm Naol</div>
+              <div>Glad to see</div>you!
+            </h1>
+          </div>
+          <section class="description">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </section>
+          <div class="text">let's connect</div>
+          <section class="button-bott">
```

Comment:
- [ ] kindly replace this section tag with ul tag as requested earlier by the previous reviewer.